enterprise/api.py

```diff
@@ -7,10 +7,19 @@
 from enterprise.models import EnterpriseCustomerBrandingConfiguration
 
 
-def get_enterprise_branding_info(provider_id=None):
+def get_enterprise_branding_info(provider_id=None, ec_uuid=None):
```

Comment:
We can also split it into two separate methods in api.
1 - `get_enterprise_branding_info_by_provider_id(provider_id=None)`
2 - `get_enterprise_branding_info_by_ec_uuid(ec_uuid=None)`